docs/editor/artificial-intelligence.md

```diff
@@ -0,0 +1,95 @@
+---
+Order: 7
+Area: editor
+TOCTitle: Refactoring
+ContentId:
+PageTitle: Code smarter with artifical intelligence
+DateApproved: 1/27/2023
+MetaDescription: Code smarter with AI-powered suggestions from GitHub Copilot.
+---
+# Code smarter with AI
```

Comment:
I wonder if we're overselling this? (Some of the suggestions aren't necessarily that smart.)
Should we pitch Copilot as help or assistance for the already smart developer?
Maybe "Coding help from AI" or "Code along with AI"?

Reply:
Just took a look at https://github.com/features/copilot/ and it feels like they are promoting Copilot as an assistant with phrases like "Your AI pair programmer" and "Don't fly solo". No mention of "smarter" or "better". They do call out "faster"

Reply:
Aligning with Copilot docs makes sense. Updated title to "Your AI pair programmer."